chart/templates/iq-server-jobs.yaml

```diff
@@ -29,6 +29,21 @@ spec:
         - name: {{ .Release.Name }}-iq-server-container
           image: {{ include "nexus-iq-server-ha.iqServerImage" . }}
           imagePullPolicy: {{ .Values.iq_server.imagePullPolicy }}
+          resources:
+            requests:
+              { { - if .Values.iq_server_jobs.resources.requests.cpu } }
```

Comment:
Spacing here looks a bit off not sure if it causes any problems other than my IDE complaining but maybe best to make it consistent with the others i.e. no spaces between `{` or `}` and no space before the first `-`

Reply:
Fixed formatting. Linting passes.